forch/forchestrator.py

```diff
@@ -9,6 +9,7 @@
 
 from google.protobuf.message import Message
 from forch.proto import faucet_event_pb2 as FaucetEvent
+from watchdog.observers import Observer
```

Comment:
Does this debian package already exist in rapture?  We should check that asap b/c if not, then I'm not sure we want to be pulling in that dependency (it's a lot of work).

Reply:
Yes, there is python3-watchdog in rapture.